brute-force-blocker-feed/info.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "brute-force-blocker-feed",
+  "version": "1.0.0",
+  "label": "BruteForceBlocker Feed",
+  "category": "Threat Intelligence",
+  "description": "BruteForceBlocker Feed it's main purpose is to block SSH bruteforce attacks via firewall.This connector facilitates automated operations related to fetching the list of IPs blocklist.<br/><br/>This connector has a dependency on the <a href=\"/content-hub/all-content/?contentType=solutionpack&amp;tag=ThreatIntelManagement\" target=\"_blank\" rel=\"noopener\">Threat Intel Management Solution Pack</a>. Install the Solution Pack before enabling ingestion of Threat Feeds from this source.",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "help_online": "",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "tags": [
+    "ThreatIntel"
+  ],
+  "ingestion_supported": true,
+  "ingestion_modes": [
+    "scheduled"
+  ],
+  "ingestion_preferences": {
+    "modules": [
+      "threat_intel_feeds",
+      "indicators"
+    ]
+  },
+  "configuration": {
+    "fields": [
+      {
+        "title": "Service",
+        "name": "service",
+        "description": "Review and select the URL of the BruteForceBlocker Feed service to which you will connect and retrieve data. You can choose https://danger.rulez.sk/projects/bruteforceblocker/blist.php.",
+        "type": "select",
+        "visible": true,
+        "editable": true,
+        "required": true,
+        "value": "BruteForceBlocker IPs Blocklist",
+        "options": [
```

Comment:
take it as a text input called server url  since this is just one input
Default value would be https://danger.rulez.sk/projects/bruteforceblocker/blist.php